dftimewolf/lib/processors/turbinia_gcp.py

```diff
@@ -149,26 +113,59 @@ def _DownloadFilesFromGCS(self, timeline_label, gs_paths):
         # Don't add a critical error for now, until we start raising errors
         # instead of returning manually each
         self.ModuleError(str(exception), critical=False)
-      self.logger.info('Downlaoded {0:s} to {1:s}'.format(path, local_path))
+      self.logger.info('Downloaded {0:s} to {1:s}'.format(path, local_path))
 
       if local_path:
         local_paths.append((timeline_label, local_path))
 
     return local_paths
 
-  def Process(self):
-    """Process files with Turbinia."""
-    log_file_path = os.path.join(self._output_path, 'turbinia.log')
-    self.logger.info('Turbinia log file: {0:s}'.format(log_file_path))
-    vm_containers = self.state.GetContainers(containers.ForensicsVM)
-    if vm_containers and not self.disk_name:
-      forensics_vm = vm_containers[0]
-      self.disk_name = forensics_vm.evidence_disk.name
-      self.logger.info(
-          'Using disk {0:s} from previous collector'.format(self.disk_name))
+  def TurbiniaSetUp(self, project, turbinia_zone, sketch_id, run_all_jobs):
+    """Sets up the object attributes.
+    Args:
+      project (str): name of the GPC project containing the disk to process.
+      turbinia_zone (str): GCP zone in which the Turbinia server is running.
+      sketch_id (int): The Timesketch sketch ID.
+      run_all_jobs (bool): Whether to run all jobs instead of a faster subset.
+    """
+    self.project = project
+    self.turbinia_zone = turbinia_zone
+    self.sketch_id = sketch_id
+    self.run_all_jobs = run_all_jobs
 
-    evidence_ = evidence.GoogleCloudDisk(
-        disk_name=self.disk_name, project=self.project, zone=self.turbinia_zone)
+    turbinia_config.LoadConfig(config_file=self.turbinia_config_file)
+    if not self.project:
+      self.project = turbinia_config.TURBINIA_PROJECT
+    if not self.turbinia_zone:
+      self.turbinia_zone = turbinia_config.TURBINIA_ZONE
+
+    if self.project is None or self.turbinia_zone is None:
+      self.ModuleError(
+          'project or turbinia_zone are not all specified, bailing out',
+          critical=True)
+      return
+
+    self.turbinia_region = turbinia_config.TURBINIA_REGION
+    self.instance = turbinia_config.PUBSUB_TOPIC
```

Comment:
I think we want `turbinia_config.INSTANCE_ID` here now.  It used to be the same as pubsub topic, and it still is often set to that, but it's a separate variable that can sometimes be different.